lib/customer.rb

```diff
@@ -0,0 +1,32 @@
+require 'csv'
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
Do `email` and `address` need to be accessors?  

Generally we give users of our class minimal access to the instance variables.  Since you don't need to give people the ability to *change* email or address, it's better to just use readers.